tests/meson.build

```diff
@@ -199,6 +199,29 @@ test(
   protocol: 'tap',
 )
 
+if meson.version().version_compare('>= 0.56.0')
```

Comment:
I don't think this check is required, since xdg-desktop-portal already depends on meson >= 0.56.2

Reply:
Took the liberty to fix this myself